qiskit/transpiler/passes/utils/check_map.py

```diff
@@ -54,7 +56,7 @@ def run(self, dag):
             physical_q0 = qubit_indices[gate.qargs[0]]
             physical_q1 = qubit_indices[gate.qargs[1]]
 
-            if self.coupling_map.distance(physical_q0, physical_q1) != 1:
+            if self.coupling_map.dist_matrix[physical_q0][physical_q1] != 1:
```

Comment:
index as tuple to avoid numpy view

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/9268df5513b4965ecf2ea457c8bf86ba697586f1